### PR TITLE
{bio}[intel/2018a] GATK v4.0.7.0 w/ Python 2.7.14 + 3.6.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/fe1c89183450ebd9fc156ad9642f846e for a full test report.

File: easybuild/easyconfigs/g/GATK/GATK-4.0.7.0-intel-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis.georgatos@uni.lu>,
+#             Kenneth Hoste (UGent)
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+##
```
Comment (verdurin) [inline easybuild/easyconfigs/g/GATK/GATK-4.0.7.0-intel-2018a-Python-3.6.4.eb]: @boegel Maybe keep the credit for Ruben on the previous version?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar03.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9a5d160c26b44135a3e61067fe309591 for a full test report.

**Comment by boegel** [comment]
@verdurin Good to go now?

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**